### PR TITLE
Set ClusterAppRunning to show the node failover destination of an app.

### PR DESCRIPTION
**Pull request by andrewd-zededa**
Find the domainmgr/DomainStatus where .DomainName matches the AppInstanceStatus.DomainName. If this is found, set the .ClusterAppRunning flag to match DomainStatus.Activated.

### CODE REVIEW

**Comment by naiming-zededa** [comment]
@andrewd-zededa I didn't realize the zedagent didn't already subscribe to the DomainStatus already, then we can actually using the AppInstanceStatus.Activated flag, which is already subscribed by the zedagent, and that will avoid this duplicate effort. also need to set the info ClusterAppRunning boolean, right?

**Comment by zedi-pramodh** [comment]
Yeah zedagent uses AppInstanceStatus, we should use that, not the domainStatus

**Comment by andrewd-zededa** [comment]
sure, I'll change it

File: pkg/pillar/cmd/zedagent/handlemetrics.go [hunk 1]
```diff
@@ -1165,6 +1165,17 @@ func PublishAppInfoToZedCloud(ctx *zedagentContext, uuid string,
...
 		}
+
+		// For Clustered apps on HV=kubevirt, update controller
+		// with the flag 'ClusterAppRunning' which designates this node
+		// as currently running the app after some failover between nodes.
+		dsItems := ctx.getconfigCtx.subDomainStatus.GetAll()
+		for _, dsItem := range dsItems {
+			ds := dsItem.(types.DomainStatus)
+			if ds.DomainName == aiStatus.DomainName {
+				ReportAppInfo.ClusterAppRunning = ds.Activated
```
Comment (zedi-pramodh) [inline pkg/pillar/cmd/zedagent/handlemetrics.go:1176]: I think this entire commit should be just one line

ReportAppInfo.ClusterAppRunning = aiStatus.Activated

Comment (andrewd-zededa) [reply pkg/pillar/cmd/zedagent/handlemetrics.go]: yeah changed it to that

**Review by zedi-pramodh — APPROVED** [review]
LGTM

**Pull request merged**